src/pages/host/getHostEventString.tsx

```diff
@@ -212,7 +212,7 @@ export const getHostEventString = (
           Assigned to run task{" "}
           <StyledLink
             data-cy="host-running-task-set-link"
-            href={`${getUiUrl()}/task${data.taskId}/${data.execution}`}
+            href={`${getUiUrl()}/task/${data.taskId}/${data.execution}`}
```

Comment:
are these links meant to point to spruce or the legacy ui?

Reply:
They currently link to legacy, but there is probably no reason for that. Should I switch them to Spruce?

Reply:
fixed